app/assets/stylesheets/application.css

```diff
@@ -1,26 +1,8 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
-
- */
-
```

Comment:
### Your Category Card
![image](https://user-images.githubusercontent.com/93039359/232198084-0de7ede1-718f-4223-9087-d8f67ba0c258.png)

### Design Category Card
![image](https://user-images.githubusercontent.com/93039359/232198220-2d66c754-a3cd-418c-a769-4663bbe5fc3b.png)


- [ ] The color of the title should not be `#0D6EFD` but it should be `#C0C0C0` :+1: Kindly do this change to make your project follow the design :blush: